draft-ietf-quic-recovery.md

```diff
@@ -59,12 +44,7 @@ normative:
 
 informative:
 
-  RFC3782:
```

Comment:
Good to know, in that case, can you add a TODO in the congestion control section that needs to be written to ensure I mention them there.